README.md

```diff
@@ -1,2 +1,114 @@
-# data-notebook-hub
-DATA-3190 Repository of Jupyter Notebooks
+
+# FEWSNET API Notebooks Repository
+
+Welcome to the FEWSNET API Notebooks Repository!
+
+This repository is a curated collection of Jupyter notebooks
+designed to serve as a practical resource for understanding
+and executing public API calls. Our notebooks provide hands-on
```

Comment:
Let's remove `public` from here and mention that we also give examples of how to make authenticated API calls.